test/integration/extensionRequests.test.js

```diff
@@ -83,13 +96,14 @@ describe("Extension Requests", function () {
     ];
 
     // Add the active task
-    taskId1 = (await tasks.updateTask(taskData[0])).taskId;
+    taskId0 = (await tasks.updateTask(taskData[0])).taskId;
+    taskId1 = (await tasks.updateTask(taskData[1])).taskId;
 
     // Add the completed task
-    taskId2 = (await tasks.updateTask(taskData[1])).taskId;
+    taskId2 = (await tasks.updateTask(taskData[2])).taskId;
 
     // Add the completed task
-    taskId3 = (await tasks.updateTask(taskData[2])).taskId;
+    taskId3 = (await tasks.updateTask(taskData[3])).taskId;
```

Comment:
won't so many await risk the chance of test timeout?

Reply:
no, haven't seen that scenario yet, as the task that we are adding are very small.